cime/config/e3sm/config_grids.xml

```diff
@@ -1023,9 +1023,9 @@
       <mask>oEC60to30v3</mask>
     </model_grid>
 
-    <model_grid alias="ne30pg2_oECv3_ICG" compset="(CAM5.+CLM.+MPASO%SPUNUP)">
+    <model_grid alias="ne30pg2_r05_oECv3_ICG" compset="(CAM5.+CLM.+MPASO%SPUNUP)">
```

Comment:
Any particular reason you chose to remove the configuration with atm and lnd on the same grid when adding the tri-grid? Just so that we don't have to support both?

Reply:
Yes, I thought for V2 we will only be supporting land on r05 and r0125 grids.